### PR TITLE
docker iac documentation  Closes #1797

### LINKED ISSUE #1797: Create Azure VM with needed software
Given that not all tools (azure-cli, azure core tools) works with the IaC, there is the opportunity to create a optional script to deploy a VM with the needed software is case is needed.

### PR DESCRIPTION
**Pull request by edwintorres**
Added docker and docker-compose to run IAC code with the version of the pre-requisite software that we know works well.

### CODE REVIEW

**Review by echappen — APPROVED** [review]
🙇 🚀

File: iac/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+RUN ./iac/install-psql.bash
+RUN ./iac/install-extensions.bash
+# Configure az cli
+# RUN az config set extension.use_dynamic_install=yes_without_prompt
+RUN az cloud set --name AzureCloud
+#Copy piipan and set the permision to execute the code
+COPY . ./piipan
+RUN chmod -R ug+rwx /piipan
+#Start the deployment with the set enviroment "tts/dev"
+CMD ["/piipan/iac/docker/start.bash", "tts/eth"]
```
Comment (kingcomma) [inline iac/Dockerfile:20]: Looks like this is referencing a local env file instead of `tts/dev`.

File: iac/docker/start.bash [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (kingcomma) [inline iac/docker/start.bash:1]:
```suggestion
#!/usr/bin/env bash
```

File: iac/Dockerfile (hunk 1 above)
Comment (edwintorres) [reply iac/Dockerfile:20]: good catch

**Comment by edwintorres** [comment]
Added docker to run iac code with the software prerequisite working version

**Pull request merged**